core/domain/config_services_test.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Testing Services for configuration properties."""
+
+import collections
+
+from core.domain import config_domain
+from core.domain import config_services
+from core.tests import test_utils
+
+
+class ConfigServicesDomainUnitTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(ConfigServicesDomainUnitTests, self).setUp()
+        ConfigObj = collections.namedtuple(
+            'ConfigObj', 'name new_config_value')
+        self.committer_id = u'configServicesCommitterID'
+        self.config_obj = ConfigObj(
+            name='propertyName', new_config_value='propertyValue')
+        self.invalid_property_name = u'_no_such_property_name'
+        self.property_dummy_value = u'TEST_VALUE'
+        self.schema_unicode = 'unicode'
+        self.cfg_schemas = config_domain.Registry.get_config_property_schemas()
+        # Get first property name, with unicode as schema type.
+        self.property_name = next(
+            x for x in self.cfg_schemas if
+            self.cfg_schemas[x]['schema']['type'] == self.schema_unicode)
+
+    def test_set_property(self):
+        with self.assertRaisesRegexp(
+            Exception, 'No config property with name.+'):
+
```

Comment:
Remove the extra lines here and below.

Reply:
About my above comment, Exception is at the same identation as the code inside "with" will be, it makes it a bit confusing & i got notification about it from my IDE.  I did "tab" to Exception line but then Lint blocked me and asked to revert as it is now in the commit.
About one line below, i placed it to address the exact issue i've talked, so Exception which is part of "with" wont be exactly above a code below, they both have same identation.
i can assign a variable with string message for Exception, this can reduce the line length and solve it all in a one liner code. what do you think?

Reply:
Oh, thanks for the explanation :)
I think it is alright for the Exception statement and the code within the with block to have the same indentation. Further, if you'd like, we can do something like [this](https://github.com/oppia/oppia/blob/develop/core/domain/classifier_domain_test.py#L112) here too.